### PR TITLE
[Overlapping] Display Spread on Strategy Card 

### LINKED ISSUE #1096: Overlapping strategy price tooltip
## Summary 

We want to expose spread information in the overlapping strategy price tooltip

## Designs

[Overlapping strategy price tooltip](https://www.figma.com/design/YpsatcY9jtxJr0SJwlFYNj/Carbon-Screen-Design?node-id=13826-49855&t=qBch34aX4mSo7iqp-4)

### How does it work?

Upon hovering over the buy or sell range of the overlapping liquidity chart:

IF strategy is "concentrated" //apply the same check we do for duplicate and edit
- add "Spread" with spread calculation
//use the spread calculation applied in the duplicate or edit flows

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
fix #1096 

Add spread on overlapping strategy card tooltip
image

### CODE REVIEW

**Review by ashachaf — APPROVED** [review]

**Review by tiagofilipenunes — CHANGES_REQUESTED** [review]
LGTM. 

Works on my strategies, strategy page, explore, with empty strategy, with offline api. E2E tests look good and spread testing is covered.

File: e2e/screenshots/strategy/recurring/Recurring limit range/create/form.png
[the remark this answers is not in the document]
Comment (GrandSchtroumpf) [reply e2e/screenshots/strategy/recurring/Recurring limit range/create/form.png:1]: hmmm the main question is why it was 1698 in the first place. The E2E specify it to be 1700...
The buy price should be 1500. I'm surprise the output is good though...
I think it's a bug unrelated to the PR

[the remark this answers is not in the document]
Comment (GrandSchtroumpf) [reply e2e/screenshots/strategy/recurring/Recurring limit range/create/form.png:1]: I created an issue to tackle this is another PR : 
https://github.com/bancorprotocol/carbon-app/issues/1364

[the remark this answers is not in the document]
Comment (tiagofilipenunes) [reply e2e/screenshots/strategy/recurring/Recurring limit range/create/form.png:1]: Works for me 👍

**Review by tiagofilipenunes — APPROVED** [review]

**Pull request merged**